tests/test_gcm.py

```diff
@@ -165,14 +151,10 @@ def config_setup(self, config: Dict[str, Any]) -> None:
             "api_key": "kii",
             "fcm_options": {"content_available": True, "mutable_content": True},
         }
-        self.service_account_file = tempfile.NamedTemporaryFile()
-        self.service_account_file.write(FAKE_SERVICE_ACCOUNT_FILE)
-        self.service_account_file.flush()
         config["apps"]["com.example.gcm.apiv1"] = {
             "type": "tests.test_gcm.TestGcmPushkin",
             "api_version": "v1",
             "project_id": "example_project",
-            "service_account_file": self.service_account_file.name,
```

Comment:
Given we still support the `service_account_file` config option, it feels odd to remove it from the tests completely.

Perhaps we should leave it as a possible variant of this class? And have a simple test which check that variant loads credentials correctly?